src/components/Header/Notifications.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const Notifications = ({ plan, notifications, emptyNotificationText }) => {
+  // check if we're always going to show notifications
+  const showNotifications = plan.isFreeAccount && notifications.length;
+  const dataCountAttr = showNotifications ? { 'data-count': notifications.length } : {};
+  return (
+    <>
+      <span className="user-menu--open active" {...dataCountAttr}>
+        <span className="ms-icon icon-notification"></span>
+      </span>
+      <div className="user-menu helper--right dp-notifications">
+        <div className="dp-msj-notif">
+          {!showNotifications ? (
+            <i>
+              {emptyNotificationText ? (
+                emptyNotificationText
+              ) : (
+                <FormattedMessage id="empty_notification_text" />
+              )}
+            </i>
+          ) : (
+            <div dangerouslySetInnerHTML={{ __html: notifications[0] }} />
```

Comment:
what is `dangerouslySetInnerHTML` ??

Reply:
Because this feature allows us to inject malicious js code.

Reply:
Oh, sorry, you are asking "what", no "why".

Here you have a little explanation: https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml
> `dangerouslySetInnerHTML` is React’s replacement for using `innerHTML` in the browser DOM. In general, setting HTML from code is risky because it’s easy to inadvertently expose your users to a _cross-site scripting (XSS) attack_. So, you can set HTML directly from React, but you have to type out `dangerouslySetInnerHTML` and pass an object with a `__html` key, to remind yourself that it’s dangerous.